init.lua

```
@@ -1,2 +1,7 @@
 -- Load the main configuration for Alex's Neovim setup
 require("alex")
+
+-- Load the color scheme...
+vim.cmd [[colorscheme moonfly]]
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)